var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -143,29 +101,6 @@ def configure_args(self):
 
         return args
 
-    def install(self, spec, prefix):
-        # Call main install method
-        super(Libfabric, self).install(spec, prefix)
-
-        # Build and install fabtests, if available
-        if not os.path.isdir('fabtests'):
-            return
-        with working_dir('fabtests'):
-            configure = Executable('./configure')
-            configure('--prefix={0}'.format(self.prefix),
-                      '--with-libfabric={0}'.format(self.prefix))
-            make()
-            make('install')
-
     def installcheck(self):
         fi_info = Executable(self.prefix.bin.fi_info)
         fi_info()
-
-        # Run fabtests test suite if available
-        if not os.path.isdir('fabtests'):
-            return
-        if self.spec.satisfies('@1.8.0,1.9.0'):
-            # make test seems broken.
-            return
-        with working_dir('fabtests'):
-            make('test')
```

Comment:
I think you missed to transfer this into the fabtests package?

Reply:
Isn't this default in an `AutotoolsPackage` in Spack?

Reply:
Ahh, yeah, I remember, there was something like that. Fine.